doc/reference/reference_lua/popen.rst

```diff
@@ -87,6 +80,10 @@ Below is a list of all ``popen`` functions and handle methods.
     | <popen-handle_fields>`               |                                 |
     +--------------------------------------+---------------------------------+
 
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+                      popen module constructors
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
```

Comment:
I think it would be good to add this constructors-methods separation into the ToC table above (the one that is in the article's body).

Reply:
Seems done, closing the discussion.